versioned_docs/version-2021sp/lecture2.md

```diff
@@ -379,15 +379,6 @@ app.listen(8080, function () {
 });
 ```
 
-Add the following to your `package.json`:
-
-```json
-// other package.json properties...
-"scripts": {
-  "start": "ts-node index.ts"
```

Comment:
We still need this section though since we want them to be able to use `yarn start` to start the app

Reply:
they'll just install `ts-node` using `yarn add --dev ts-node` instead of with a global flag

Reply:
Oke, I'll add it back in :,)